lib/rspec/rails/configuration.rb

```diff
@@ -48,6 +48,15 @@ def self.initialize_configuration(config)
       config.include RSpec::Rails::ViewExampleGroup,       :type => :view
       config.include RSpec::Rails::FeatureExampleGroup,    :type => :feature
 
+      config.before :suite do
+        return unless defined?(ActiveRecord) && defined?(::RSpec::Mocks)
```

Comment:
`return` can only be used within a method that hasn't yet exited, and that's not the case here -- `initialize_configuration` has exited long before this hook runs.   In this situation, it triggers the `LocalJumpError` you are seeing on travis.  You can use `break` to exit the block instead.